### DISCUSSION TITLE
Configurations of OpenMPTCPRouter

### DISCUSSION THREAD

**Discussion by techworldthink** [Ysurac/openmptcprouter#1789] [Q&A] [answered]
How to reset settings to default ?

**Comment by Adorfer** [comment, marked as answer]
if it's an image on squashfs the command on the shell is "firstboot" and confirm with "y" and then "reboot".

**Reply by techworldthink** [reply]
In my experiment the network speed did not increase . How to understand the reason for that?

**Reply by Ysurac** [reply]
All must be green in System->OpenMPTCProuter status page.
If it's ok, there is many cause that can be a problem : slow router or VPS CPU, wrong configuration, ISP that block MPTCP,... but to check that it's better to open an issue and use the template to give all needed info.

**Comment by Adorfer** [comment]
alternativly you can reset the configuration by flashing a sysupgrade-Image, either a new vesion or just the same one and tick the option "do **not keep** settings" in the UI or use on the console "sysupgrade -n" with the image name.
